src/main/java/frc/robot/subsystems/WristIntake.java

```diff
@@ -19,18 +18,13 @@
  */
 public class WristIntake extends SubsystemBase {
 
-    private final CANSparkMax leftWristMotor =
-        new CANSparkMax(Constants.Wrist.LEFT_MOTOR_ID, MotorType.kBrushless);
-    private final CANSparkMax rightWristMotor =
-        new CANSparkMax(Constants.Wrist.RIGHT_MOTOR_ID, MotorType.kBrushless);
-    private final MotorControllerGroup wristIntakeMotors =
-        new MotorControllerGroup(leftWristMotor, rightWristMotor);
-    private final DoubleSolenoid wristSolenoid;
+    private final WPI_TalonFX wristIntakeMotor = new WPI_TalonFX(Constants.Wrist.WRIST_INTAKE_ID);
 
 
     private final DigitalInput coneSensor1 = new DigitalInput(Constants.Wrist.CONE_SENSOR_ID);
     private final DigitalInput cubeSensor1 = new DigitalInput(Constants.Wrist.CUBE_SENSOR_ID_LEFT);
     private final DigitalInput cubeSensor2 = new DigitalInput(Constants.Wrist.CUBE_SENSOR_ID_RIGHT);
```

Comment:
Did u guys add touch sensors back? if not i'd remove the references to them as they will continue to trigger the LEDs

Reply:
Removed